js/apps/system/_admin/aardvark/APP/react/src/components/form/FormikFormControl.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  FormControl,
+  FormControlProps,
+  FormErrorMessage,
+  FormErrorMessageProps,
+  FormLabel,
+  FormLabelProps
+} from "@chakra-ui/react";
+import { useField } from "formik";
+import React from "react";
+
+export type BaseFormControlProps = FormControlProps & {
+  name: string;
+  labelProps?: FormLabelProps;
+  errorMessageProps?: FormErrorMessageProps;
+};
+export const FormikFormControl = ({
```

Comment:
Typescript may well handle this, just as it can handle any poorly written code as long as it compiles. My point was about this being more in line with conventions followed in most flagship modules, to make it easier for other developers to follow and use.